### ISSUE TITLE
Intimidation rates may be incorrect.

### ISSUE THREAD

**Issue by muneonna** [EdenServer/community#4901]
### Checklist

- [x ] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x ] I have provided reproducible steps. (No "as the title says" posts please. Provide reproducible steps even if it seems like a waste of time.)
- [x ] I have provided my client version in the details. (type /ver into your game window)
- [x ] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details

Client Version 30181205_0

It appears that something may be incorrect about intimidation rates. 

Firstly, I have the understanding that these values are are stated as a fraction over 1024, IE BST +10 trait is "100/1024", and each bonus is added according to this. I'm not certain how true this is, but I believe it is the case with other things, such as haste, and I feel it would make sense here as well. If this is incorrect, sorry. That said, the values would still seem even more off without it.

As BST75, I went to Delk. Tower, first floor where there are three kinds of bats. Seeker bats, Ancient Bat, and Giga's Bat.

Applicable gear and merits: Suzaku's Scythe, Tamer's Ring, Beast Trousers. I was also wearing Bibiki Seashell, but that shouldn't be applicable here. I have 2 merits in Killer effects. BST passive should offer +10 Bird Killer. 

Per the following links/examples, I believe I should be seeing aprox. 20% killer effect proc. I realize this isnt the max, but it was a nice round number to play with.

(Suzaku's Scythe) +4 https://www.bg-wiki.com/ffxi/Suzaku%27s_Scythe
(Beast Trousers) +1 https://www.bluegartr.com/threads/103597-Temporary-Random-Facts-Thread?p=4641701&viewfull=1#post4641701
(Tamer's Ring) +3 - Active while HP ≤ 75% and TP ≤ 100%. I made sure to only test below these values, with a double-check on the 4th test.
(Killer Merits) +2
(Passive BST) +10

With that in mind, I should be seeing 200/1024, or approximately 19.53%

Because I became aware my TP gain may have skewed the results for the ring after I began, I attempted a 4th test on a pair of Seeker Bats (included, named as the fourth test.). These results were still "off" by more than a typical margin of error.

[CLEAN Giga Bat - Muneonna_2022.02.18.log](https://github.com/EdenServer/community/files/8100325/CLEAN.Giga.Bat.-.Muneonna_2022.02.18.log)

  TLDR; 7.23%

[CLEAN Seeker Bats - Muneonna_2022.02.18.log](https://github.com/EdenServer/community/files/8100326/CLEAN.Seeker.Bats.-.Muneonna_2022.02.18.log)

  TLDR; 14.52%

[CLEAN Ancient Bat - Muneonna_2022.02.18.log](https://github.com/EdenServer/community/files/8100329/CLEAN.Ancient.Bat.-.Muneonna_2022.02.18.log)

  TLDR; 12.79%

[CLEAN x2 Seeker Bats 4th test - Muneonna_2022.02.18.log](https://github.com/EdenServer/community/files/8100328/CLEAN.x2.Seeker.Bats.4th.test.-.Muneonna_2022.02.18.log)

  TLDR; 15.31%

As a final note, I also did a test with zero "killer gear", vs x2 Seeker Bats. 

This would be the base +10 plus my x2 merits for a total of Bird Killer +12. If my (well, dubious?) assertion of 120/1024 holds, I should see an 11.72% killer proc rate.

[CLEAN x2 Seeker Bats NO Killer Gear - Muneonna_2022.02.18.log](https://github.com/EdenServer/community/files/8100469/CLEAN.x2.Seeker.Bats.NO.Killer.Gear.-.Muneonna_2022.02.18.log)

  TLDR; 7.90%
  
  The conclusion I'm drawing from this seems to be that Eden is only providing BST a +5 bonus to killer instead of the appropriate 10. I haven't found any media stating killer effects were increased following an update.

**Comment by muneonna** [comment]
Its also possible that something completely different is goin on with the Giant's pet bat, since its numbers seemed to imply it was ignoring all my gear's additional killer stats.

**Comment by 59blargedy** [comment]
thank you for the report, just want to clarify something - at an admittedly quick glance, i don't see where passive trait for bird killer is 10 until level 79. can you link the source that shows 10 instead of 8?

**Comment by muneonna** [comment]
I have found on https://www.bg-wiki.com/ffxi/Bird_Killer (probably the same place you did?) that it seems someone has posted as fact that there was a bird killer II trait later introduced. 

At least as thats concerned, certainly my numbers may also be off by 2% (I found that after I posted, because of course I did)

Though I wasn't actually able to confirm that anyone knew what the actual numbers were back then? At least, not until after 2011.

[CLEAN x4 Akbaba - Muneonna_2022.02.18.log](https://github.com/EdenServer/community/files/8100933/CLEAN.x4.Akbaba.-.Muneonna_2022.02.18.log)

Here is one more, larger test. Went and pulled four Akbaba, this time without the (frankly, troublesome) Tamer's ring.

TLDR; 11.58% (Suzaku, Beast pants, x2 merit) I believe that should be 15, if killer is only 8 in era?

**Comment by muneonna** [comment]
I was having trouble finding where I found my assertion that it was 10% - but here it is.

https://www.bg-wiki.com/index.php?title=Bird_Killer&direction=next&oldid=28740

It IS from 2011, but there were no numbers posted prior to that, near as I can tell.

**Comment by muneonna** [comment]
(its the merits isnt it. MY MERITS DO NOTHING haha)

Ok, thanks!

**Comment by 59blargedy** [comment]
resolved in April 18, 2022 patch
